starknet_py/net/client_models.py

```diff
@@ -203,16 +203,41 @@ class InvokeTransaction(DeprecatedTransaction):
     nonce: Optional[int] = None
```

Comment:
Please also remove the old classes `InvokeTransaction`, `DeclareTransaction` and `DeployAccountTransaction`.